src/frontend-build.sh

```diff
@@ -3,8 +3,10 @@
 # --------------------------------------------
 # Export vars for helper scripts to use
 # --------------------------------------------
-export APP_NAME=$(node -e "console.log(require(\"${WORKSPACE:-.}${APP_DIR:-}/package.json\").insights.appname)")
-export CONTAINER_NAME="$APP_NAME-build-main"
+# Flexibilty to provide APP_NAME from pr_check.sh parent file.
+export APP_NAME=${APP_NAME:=$(node -e "console.log(require(\"${NODE_ROOT}/package.json\").insights.appname)")}
```

Comment:
MAS Jenkins instances doesn;t have node installed. so the pipeline fails at this step.